### ISSUE TITLE
Fix Composer 2 compatibility

### ISSUE THREAD

**Issue by brandonkelly** [a-digital/critical-css#1]
When installing this plugin with Composer 1, the following deprecation notice is output:

> Deprecation Notice: Class `adigital\criticalcss\assetbundles\CriticalCss\CriticalCssAsset` located in `./vendor/adigital/critical-css/src/assetbundles/criticalcss/CriticalCssAsset.php` does not comply with psr-4 autoloading standard\. It will not autoload anymore in Composer v2\.0\.

Composer 2 was released on October 24, and is now the default version that will be installed, so it’s critical that this gets resolved ASAP, or people will start getting `Class not found` errors.

To fix, run the following command:

```bash
> git mv src/assetbundles/criticalcss src/assetbundles/CriticalCss
```

Then commit your change and [tag a new release](https://craftcms.com/docs/3.x/extend/plugin-store.html#plugin-releases).

_(Note that you must use the `git mv` command, as Git tends to not notice case-sensitive file renames otherwise.)_

**Comment by matt-adigital** [comment]
Thanks Brandon, fixed in release 1.0.3

**Comment by brandonkelly** [comment]
Thanks :)
